### PR TITLE
Added Aliases to Characters and Tags for Popular Squads

### PR DESCRIPTION
**Pull request by CaptanHamm**

### CODE REVIEW

File: src/constants/characters.js [hunk 1]
```diff
@@ -396,7 +396,7 @@ let charactersArray = [
       0, // crit avoidance
       false
     ),
-    ['Light Side', 'Ewok', 'Support']
+    ['Light Side', 'Ewok', 'Support', 'chirp', 'Murderbears']
```
Comment (grandivory) [inline src/constants/characters.js]: "chirp" will already work when searching for Chief Chirpa

File: src/constants/characters.js [hunk 2]
```diff
@@ -415,7 +415,7 @@ let charactersArray = [
       0, // crit avoidance
       false
     ),
-    ['Light Side', 'Jawa', 'Tank']
+    ['Light Side', 'Jawa', 'Tank', 'neb']
```
Comment (grandivory) [inline src/constants/characters.js]: "neb" will already work when searching for Chief Nebit

File: src/constants/characters.js [hunk 3]
```diff
@@ -1819,7 +1819,7 @@ let charactersArray = [
       0, // crit avoidance
       false
     ),
-    ['Light Side', 'Galactic Republic', 'Jedi', 'Healer']
+    ['Light Side', 'Galactic Republic', 'Jedi', 'Healer', 'lumi']
```
Comment (grandivory) [inline src/constants/characters.js]: "lumi" will already work when searching for Luminara

File: src/constants/characters.js [hunk 4]
```diff
@@ -2006,7 +2006,7 @@ let charactersArray = [
       0, // crit avoidance
       false
     ),
-    ['Light Side', 'Rebel', 'Jedi', 'Tank']
+    ['Light Side', 'Rebel', 'Jedi', 'Tank', 'ben', 'OB']
```
Comment (grandivory) [inline src/constants/characters.js]: "ben" will already work when searching for Obi-wan

File: src/constants/characters.js [hunk 5]
```diff
@@ -2695,7 +2695,7 @@ let charactersArray = [
       0, // crit avoidance
       false
     ),
-    ['Light Side', 'Scoundrel', 'Attacker']
+    ['Light Side', 'Scoundrel', 'Attacker', 'Vets', 'Vet']
```
Comment (grandivory) [inline src/constants/characters.js]: "vet" will already work when searching for the veterans

File: src/constants/characters.js [hunk 6]
```diff
@@ -2714,7 +2714,7 @@ let charactersArray = [
...
```
Comment (grandivory) [inline src/constants/characters.js]: "vet" will already work when searching for the veterans

**Comment by CaptanHamm** [comment]
removed unneeded aliases.

**Pull request merged**